testkit/src/main/java/org/apache/calcite/test/catalog/MockCatalogReaderSimple.java

```diff
@@ -291,13 +297,10 @@ protected MockCatalogReaderSimple(RelDataTypeFactory typeFactory,
     registerTable(suppliersTable);
 
     // Register "EMP_20" and "EMPNULLABLES_20 views.
-    // Same columns as "EMP" amd "EMPNULLABLES",
-    // but "DEPTNO" not visible and set to 20 by default
-    // and "SAL" is visible but must be greater than 1000,
-    // which is the equivalent of:
+    // Same columns as "EMP" amd "EMPNULLABLES", but "DEPTNO" not visible and set to 20 by default
+    // and "SAL" is visible but must be greater than 1000, which is the equivalent of:
```

Comment:
Is it necessary to do this comment reformatting?

Reply:
Without the comment reformatting I get the following test failure:
```
/home/runner/work/calcite/calcite/testkit/src/main/java/org/apache/calcite/test/catalog/MockCatalogReaderSimple.java:79:3: Method length is 376 lines (max allowed is 370). [MethodLength]
```
This seemed like the simplest change, but I'm happy to do something else instead.

Reply:
I see, then we'd better split this method into smaller ones in a separate issue, and I'm ok for this workaround in this one.

Reply:
I've logged CALCITE-5574 for this.